hs_core/models.py

```diff
@@ -1769,8 +1769,16 @@ class Meta:
         abstract = True
         unique_together = ("content_type", "object_id")
 
-    def check_irods_sync(self, raise_exceptions=False):
-        """ Check whether files in self.files and on iRODS agree """
+    def check_irods_files(self, stop_on_error=False, log_errors=True,
+                          echo_errors=False, return_errors=False):
+        """
+        Check whether files in self.files and on iRODS agree
```

Comment:
So the check is only in one direction - from Django to iRODS. Should we be checking in the reverse direction?